go-apps/meep-wais/sbi/wais-sbi.go

```diff
@@ -195,8 +223,17 @@ func processActiveScenarioUpdate() {
 	sbi.activeModel.UpdateScenario()
 
 	scenarioName := sbi.activeModel.GetScenarioName()
-	sbi.updateScenarioNameCB(scenarioName)
-
+	var update bool
+	sbi.updateScenarioNameCB(scenarioName, &update)
+
+	// Connect to Metric Store
+	if update {
+		var err error
+		metricStore, err = met.NewMetricStore(scenarioName, sbi.sandboxName, influxAddr, redisAddr)
+		if err != nil {
+			log.Error("Failed connection to metric-store: ", err)
+		}
+	}
```

Comment:
The SBI should not depend on the WAIS to determine when to connect to the metric store.

Reply:
i put all the logic in sbi now